packages/fxa-settings/src/lib/metrics.ts

```diff
@@ -269,6 +269,7 @@ export function useViewEvent(
 ) {
   useEffect(() => {
     logViewEvent(viewName, eventName, eventProperties);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, dependencies);
```

Comment:
The warning that prompted me to add this:

> React Hook useEffect was passed a dependency list that is not an array literal. This means we can't statically verify whether you've passed the correct dependencies

I actually don't understand this warning, because if you look a little further up the default value for this _is_ an array literal...

Reply:
it's an array variable but not a literal. the warning is that it can't statically analyze the values of `dependencies`